basxconnect/core/views/person/person_search_views.py

```diff
@@ -30,7 +30,54 @@ def searchperson(request):
         .filter_or(personnumber=query)
     )
 
-    ret = display_results(objects, highlight)
+    def onclick(person):
+        link = reverse_model(
+            person,
+            "edit",
+            kwargs={"pk": person.pk},
+        )
+        return f"document.location = '{link}'"
+
+    ret = _display_results(
+        objects,
+        highlight,
+        onclick,
+    )
+
+    return HttpResponse(
+        hg.DIV(
+            ret,
+            _class="raised",
+            style="margin-bottom: 1rem; padding: 16px 0 48px 48px; background-color: #fff",
+        ).render({})
+    )
+
+
+# Search view
+# simple person search view, for use with ajax calls
+def searchperson_and_insert(request):
+    query = request.GET.get("q")
+    selected_result_selector = request.GET.get("target-id-to-store-selected")
```

Comment:
I think passing a selector via GET query is not an ideal solution. But changing that would be too much work for now I think, we can still do some refactoring when we do the next iteration.

Reply:
will take care of it. resolving this conversation as we have another one open regarding the bigger issue.